src/GitHub/Data/PullRequests.hs

```diff
@@ -74,12 +77,10 @@ data PullRequest = PullRequest
     , pullRequestComments       :: !Count
     , pullRequestDeletions      :: !Count
     , pullRequestAdditions      :: !Count
-    , pullRequestReviewComments :: !Count
     , pullRequestBase           :: !PullRequestCommit
     , pullRequestCommits        :: !Count
     , pullRequestMerged         :: !Bool
     , pullRequestMergeable      :: !(Maybe Bool)
-    , pullRequestMergeableState :: !MergeableState
```

Comment:
ah sorry, that was due to my confusion between pull request / pull request event. I'll put it back in